src/core/metrics/sources/nginx_plus.go

```diff
@@ -608,6 +629,39 @@ func (c *NginxPlus) streamUpstreamMetrics(stats, prevStats *plusclient.Stats) []
 	return upstreamMetrics
 }
 
+func getTimeMetrics(times []float64, metricType string) float64 {
```

Comment:
this function is very similar to the `getTimeMetricsMap` function here https://github.com/nginx/agent/blob/main/src/core/metrics/sources/nginx_access_log.go#L430

Is there a way that we could refactor these functions into a common function and move it to metrics_util.go ?